idd/CMakeLists.txt

```diff
@@ -1,14 +1,14 @@
-install(FILES "V7-2-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-0-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-1-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-2-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-3-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-4-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-5-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-6-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-7-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-8-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
-install(FILES "V8-9-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V7-2-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-0-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-1-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-2-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-3-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-4-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-5-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-6-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-7-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-8-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
+#install(FILES "V8-9-0-Energy+.idd" DESTINATION "PreProcess/IDFVersionUpdater")
```

Comment:
Not installing the super old IDDs anymore.  We can make sure these are available for the rare user who needs them, but cleaning up the old versions in the installer should save download time and disk space for like 99% of the users.